### PR TITLE
Add `CopyCard` action

### LINKED ISSUE #1294: Card: Copy

### PR DESCRIPTION
**Pull request by lvfxx**
Closes #1294

### CODE REVIEW

File: plugins/board-resources/src/components/popups/CopyCard.svelte [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  let inputRef: TextArea
+  let title = object.title
+  let status: Status = OK
+  const selected = {
+    space: object.space,
+    state: object.state,
+    rank: object.rank
+  }
+
+  async function copyCard() {
```
Comment (haiodo) [inline plugins/board-resources/src/components/popups/CopyCard.svelte]: eslint indentation is missing

Comment (haiodo) [inline plugins/board-resources/src/components/popups/CopyCard.svelte]: eslint indentation is missing, return type is missing

**Review by haiodo — APPROVED** [review]

File: plugins/board-resources/src/components/popups/CopyCard.svelte (hunk 1 above)
Comment (lvfxx) [reply plugins/board-resources/src/components/popups/CopyCard.svelte]: Fixed.

File: plugins/board-resources/src/components/popups/CopyCard.svelte [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    const incResult = await client.update(sequence, { $inc: { sequence: 1 } }, true)
+
+    const value: AttachedData<Card> = {
+      state: selected.state,
+      doneState: null,
+      number: (incResult as any).object.sequence,
+      title: title,
+      rank: selected.rank,
```
Comment (haiodo) [inline plugins/board-resources/src/components/popups/CopyCard.svelte:46]: I think it should be new rank between Target item and previous in new list.

**Pull request merged**